packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-color/gux-form-field-color.tsx

```diff
@@ -93,6 +93,7 @@ export class GuxFormFieldColor {
           required={this.required}
         >
           <slot name="label" onSlotchange={() => this.setLabel()} />
+          <slot name="tooltip" />
```

Comment:
Should there be a check on this slot to make sure it only contains an icon-tooltip?

Reply:
yes and maybe it should be called something like `additional-info`